packages/babel-register/src/cache.js

```diff
@@ -49,14 +49,14 @@ export function save() {
       case "EACCES":
       case "EPERM":
         console.warn(
-          `Babel could not write cache to file: ${FILENAME} 
+          `Babel could not write cache to file: ${FILENAME}
 due to a permission issue. Cache is disabled.`,
         );
         cacheDisabled = true;
         break;
       case "EROFS":
         console.warn(
-          `Babel could not write cache to file: ${FILENAME} 
+          `Babel could not write cache to file: ${FILENAME}
```

Comment:
The whitespace here is intentional, otherwise there is no whitespace between `filename` and "because".

Reply:
Oh let me fix that I think that was the linter acting up